### PR TITLE
converted tests to pytest

### PR DESCRIPTION
**Pull request by OliverW10**
addresses #1

### CODE REVIEW

**Comment by james-ward** [comment]
closes #1

**Review by james-ward — APPROVED** [review]
lgtm

File: test_images.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# TODO: could calculate the allowed error based on what would result in a shot missing 
+allowed_azimuth_error = math.radians(5) 
+allowed_dist_error = 0.3
+
+@pytest.mark.parametrize("filename,expected", images)
+def test_sample_images(filename: str, expected: Tuple[float, float]):
+    vision = Vision(MockCameraManager(filename))
+    outputs = (41.99, 1.9)#vision.run()
+    error = [abs(output-expected) for output, expected in zip(outputs, expected)]
```
Comment (auscompgeek) [inline test_images.py:23]: This line confused me a fair bit. Would it be better to unpack `expected` and `outputs`, so the maths is clearer?

**Review by auscompgeek — COMMENTED** [review]
My review is non-blocking.

File: test_images.py (hunk 1 above)
Comment (OliverW10) [reply test_images.py:23]: do you mean change `error` into two variables rather than a list ?

**Pull request merged**